### PR TITLE
feat(discover): Add snql support for epm and eps

### PR DESCRIPTION
**Pull request by edwardgou-sentry**
Adds SnQL support to epm and eps functions with tests

### CODE REVIEW

File: src/sentry/search/events/fields.py [hunk 1]
```diff
@@ -2385,9 +2385,19 @@ def __init__(self, dataset: Dataset, params: ParamsType):
                         alias,
                     ),
                 ),
+                SnQLFunction(
+                    "eps",
+                    snql_aggregate=lambda args, alias: self._resolve_eps(args, alias),
```
Comment (Zylphrex) [inline src/sentry/search/events/fields.py:2390]: This looks redundant as `snql_aggregate=self_resolve_eps` should suffice. Additionally, is there any reason not to just inline these with the lambda? They look simple enough that we should just inline them.

Comment (edwardgou-sentry) [reply src/sentry/search/events/fields.py]: Hey Tony, good point. No particular reason not to inline. Committed this update!

File: src/sentry/search/events/fields.py [hunk 2]
```diff
@@ -2385,9 +2385,25 @@ def __init__(self, dataset: Dataset, params: ParamsType):
...
+                SnQLFunction(
+                    "epm",
+                    snql_aggregate=lambda args, alias: Function(
+                        "divide",
+                        [Function("count", []), Function("divide", [args["interval"], 60])],
+                        alias,
+                    ),
+                    optional_args=[IntervalDefault("interval", 1, None)],
+                    default_result_type="number",
+                ),
```
Comment (wmak) [inline src/sentry/search/events/fields.py:2405]: Could you add the `tpm`&`tps` aliases as well? just setting them in the dictionary to map to epm&eps will suffice (see how we did it with `FUNCTION_ALIASES` before)

Comment (edwardgou-sentry) [reply src/sentry/search/events/fields.py:2405]: Added!

File: src/sentry/search/events/fields.py [hunk 3]
```diff
@@ -2420,6 +2436,9 @@ def __init__(self, dataset: Dataset, params: ParamsType):
                 ),
             ]
         }
+        # In Performance TPM is used as an alias to EPM
```
Comment (wmak) [inline src/sentry/search/events/fields.py:2439]: nit: I don't think this comment makes sense here, could we also move `FUNCTION_ALIASES` to search/events/constants?

**Review by wmak — APPROVED** [review]
smol nit, otherwise lgtm

**Pull request merged**